dataland-community-manager/src/test/kotlin/org/dataland/datalandcommunitymanager/controller/CompanyRightsControllerTest.kt

```diff
@@ -0,0 +1,194 @@
+package org.dataland.datalandcommunitymanager.controller
+
+import org.dataland.datalandbackend.openApiClient.api.CompanyDataControllerApi
+import org.dataland.datalandbackend.openApiClient.infrastructure.ClientException
+import org.dataland.datalandbackendutils.utils.JsonUtils.defaultObjectMapper
+import org.dataland.datalandcommunitymanager.DatalandCommunityManager
+import org.dataland.datalandcommunitymanager.model.companyRights.CompanyRight
+import org.dataland.datalandcommunitymanager.model.companyRights.CompanyRightAssignment
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.kotlin.doThrow
+import org.mockito.kotlin.whenever
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.jdbc.EmbeddedDatabaseConnection
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.http.MediaType
+import org.springframework.security.test.context.support.WithMockUser
+import org.springframework.test.context.bean.override.mockito.MockitoBean
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.ResultMatcher
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers
+import java.util.UUID
+
+@SpringBootTest(classes = [DatalandCommunityManager::class], properties = ["spring.profiles.active=nodb"])
+@AutoConfigureTestDatabase(connection = EmbeddedDatabaseConnection.H2)
+@AutoConfigureMockMvc
+@ExtendWith(SpringExtension::class)
+class CompanyRightsControllerTest
+    @Autowired
+    constructor(
+        val mockMvc: MockMvc,
```

Comment:
```suggestion
        private val mockMvc: MockMvc,
```
Would it be better to make this private?